common/src/main/java/nl/tudelft/trustchain/common/bitcoin/WalletService.kt

```diff
@@ -0,0 +1,110 @@
+package nl.tudelft.trustchain.common.bitcoin
+
+import android.util.Log
+import com.google.common.util.concurrent.Service
+import nl.tudelft.ipv8.keyvault.PublicKey
+import nl.tudelft.trustchain.common.eurotoken.TransactionRepository
+import org.bitcoinj.core.ECKey
+import org.bitcoinj.core.PeerAddress
+import org.bitcoinj.kits.WalletAppKit
+import org.bitcoinj.params.RegTestParams
+import java.io.File
+import java.net.InetAddress
+import java.net.URL
+
+class WalletService {
+
+    companion object {
+        private lateinit var globalWallet: WalletAppKit
+        private val bitcoinFaucetEndpoint = "http://134.122.59.107:3000"
```

Comment:
It'd better to extract out this endpoint into config (or even to gradle config) instead of hardcoding here.

Reply:
I have moved the endpoint to the build.grade